Contents/Code/__init__.py

```diff
@@ -697,9 +699,8 @@ def getImagesFromASS(self, metadata, media, tvdbid, movie, num=0):
     entry = postersXml.xpath("/tvdb4entries/posters[@tvdbid='%s']" % tvdbid)
     if not entry: Log.Error("tvdbid '%s' is not found in xml file" % tvdbid); return
     for line in filter(None, entry[0].text.strip().replace("\r","\n").split("\n")):
-      num += 1; seasonposternum += 1
-      season, posterURL = line.strip().split("|",1); season = str(int(season))
-      posterPath = "seasons/%s-%s-%s" % (tvdbid, season, os.path.basename(posterURL))
+      season, posterURL, num, seasonposternum = str(int(line.strip().split("|",1))), num+1, seasonposternum+1 #str(int(x)) remove leading 0 from number string
```

Comment:
This will have issues for the same reason as stated in https://github.com/ZeroQI/Hama.bundle/pull/71/files#r72722697